interface/reports/receipts_by_method_report.php

```diff
@@ -523,12 +523,13 @@ function sel_procedure() {
         $sqlBindArray = array();
         $query = "SELECT a.pid, a.encounter, a.post_time, a.pay_amount, " .
           "a.adj_amount, a.memo, a.session_id, a.code, a.payer_type, fe.id, fe.date, " .
-          "fe.invoice_refno, s.deposit_date, s.payer_id, s.reference, s.payment_method, i.name " .
+          "fe.invoice_refno, s.deposit_date, s.payer_id, s.reference, s.payment_method, i.name, l.option_id, l.title " .
           "FROM ar_activity AS a " .
           "JOIN form_encounter AS fe ON fe.pid = a.pid AND fe.encounter = a.encounter " .
           "JOIN forms AS f ON f.pid = a.pid AND f.encounter = a.encounter AND f.formdir = 'newpatient' " .
           "LEFT JOIN ar_session AS s ON s.session_id = a.session_id " .
           "LEFT JOIN insurance_companies AS i ON i.id = s.payer_id " .
+          "LEFT JOIN list_options AS l ON l.option_id = s.payment_method " .
           "WHERE ( a.pay_amount != 0 OR a.adj_amount != 0 )";
```

Comment:
hi @TheYairDa , thanks for the pull request, you'll need to tie list_id to payment_method or else you'll grab entries meant for the price_level